README.md

```diff
@@ -1,24 +1,80 @@
-**STILL IN DEVELOPMENT, NOT FIT FOR GENERAL CONSUMPTION, DO NOT USE IF NOT A DEVELOPER!**
-
 # Slic3r plugin for OctoPrint
 
-**WARNING** Not functional yet since slic3r's return codes are behaving unexpectedly. WIP for others to take a look at
```

Comment:
I tried yesterday to slice in Windows (I don't have the change of testing under Mac OS) running slic3r-console and loading the profile file created by the plugin when doing the tests under the Raspberry Pi (that is, without installing OctoPrint but using Slic3r as I believe the plugin would do). I with with 10-15 different stl files and the results were positive for version 1.2.9 but didn't work for 1.1.7 (it returned and error related with a library and stopped) as long as the file has a mainfold. I'll update the README file showing the warning.
I understand that this might take a while as there are other priorities (good luck with it by the way).